### DISCUSSION TITLE
Issue with Github Desktop

### DISCUSSION THREAD

**Discussion by PiccolinGiulio** [community/community#55953] [Projects and Issues]
### Select Topic Area

Question

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

Hi, I'm facing an issue with Github Desktop: when I try to sign in I get the following screen:
<img width="344" alt="Cattura" src="https://github.com/community/community/assets/96427699/963a7f9d-1376-4a90-abe4-32824a33cc85">
If I click on "Continue with browser" the app starts an endless loading. I've tried many browsers and also to uninstall and reinstall but nothing seems to work.
Thanks in advance for the help.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by ryanlua** [comment]
Hi @this00,

There might be other factors or problems that could be affecting GitHub Desktop and your ability to sign in. Can you make sure of the following:

* **Temporary disable firewall or antivirus software.** Your firewall or antivirus could be flagging GitHub Desktop as malicious.

* **Update GitHub Desktop.** You could be using an older version of GitHub Desktop so you should check if it needs an update. 

* **Clear app data and cache** Uninstalling could leave some app data and cache behind. You can go to `%APPDATA%\GitHubDesktop` and delete that folder before installing GitHub Desktop again.

Let me know if you still have the problem or this has fixed it.

**Reply by PiccolinGiulio** [reply]
Thank you for the answer. I've done all the steps you suggested, but I still can't log in.

**Reply by ryanlua** [reply]
Can I ask what browser you are using and if you can change the default browser to another one temporarily to see if it solves the issue?

**Reply by PiccolinGiulio** [reply]
I've tried Chrome, Edge, Firefox and Opera: I don't know why, but only Opera works. Thank you for the help.

**Reply by ryanlua** [reply]
That's quite odd but glad to help you. You can mark this thread as answered if you believe your issue is solved.
